pyproject.toml

```diff
@@ -59,6 +59,9 @@ MarkupSafe = "2.1.5"
 Sphinx = "^5.3.0"
 poethepoet = "^0.19.0"
 
+[tool.poetry.group.dev.dependencies]
+packaging = "^24.1"
```

Comment:
where is the `packaging` module used? why do we need a new dev-dependency ?

Reply:
here is a commit without this dependency. feel free to pick it up: https://github.com/XRPLF/xrpl-py/compare/test-branch-builds...ckeshava:xrpl-py:py38-updates?expand=1

Reply:
I couldn't find the usages of `packaging` module outside of the poetry.lock file

Reply:
This can be removed. It was automatically added when I updated the Python version.